Kuroba/app/src/main/java/com/github/adamantcheese/chan/utils/StringUtils.java

```diff
@@ -90,4 +90,31 @@ public static String decodeBase64(String base64Encoded) {
 
         return bytesToHex(bytes);
     }
+
+    public static String maskPostNo(int postNo) {
+        String postNoString = String.valueOf(postNo);
+        if (postNoString.length() >= 4) {
+            return postNoString.substring(0, postNoString.length() - 3) + "XXX";
+        }
+
+        return postNoString;
+    }
+
+    public static String maskUrl(@NonNull String url) {
+        if (url.length() < 4) {
```

Comment:
If the URL is less than 4 characters, it's not a URL. `http://` is the bare minimum you need for a URL, and that's not even counting the actual domain and whatnot. Kinda worthless check.

Reply:
It's there just to avoid possible OOBEs. I just don't want to convert it into HttpUrl to check whether it is an actual url or not because that will create unnecessary allocation spam.